test/cmdlineTests.sh

```diff
@@ -41,6 +41,12 @@ case "$(uname)" in
         SOLC="$REPO_ROOT/${SOLIDITY_BUILD_DIR}/solc/solc";;
 esac
 
+UNIX_TO_DOS=""
```

Comment:
I'm not sure we should introduce this complication and would prefer the more sloppy comparison.

Reply:
OK, reverted for now. We can implement flexible comparison (maybe optional?).
It could be taken together with the issue that our tests are insensitive to extra trailing newlines.